config/default.yml

```
@@ -2612,6 +2612,13 @@ Lint/UselessConstantScoping:
   Enabled: pending
   VersionAdded: '1.72'
 
+Lint/UselessDefaultValueArgument:
+  Description: 'Checks for usage of `fetch` or `Array.new` with default value argument and block.'
+  Enabled: pending
+  VersionAdded: '<<next>>'
+  SafeAutoCorrect: false
```

With this approach, which detects cases where the receiver is a variable, `Safe: false` applies. Personally, I'm still concerned about the risk of false positives, which remains a trade-off against catching false negatives.

```suggestion
  Safe: false
```

---

Updated